src/library/Uno.Themes.WinUI.Markup/Themes/FloatingActionButton.cs

```diff
@@ -0,0 +1,270 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class FloatingActionButton
+		{
+			public static class Resources
+			{
+				public static class Default
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabForeground")]
+						public static ResourceValue<Brush> Default => new("FabForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabBackground")]
+						public static ResourceValue<Brush> Default => new("FabBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabBackgroundDisabled", true);
+					}
+				}
+
+				public static class StateOverlay
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabStateOverlayBackground")]
+						public static ResourceValue<Brush> Default => new("FabStateOverlayBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabStateOverlayBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabStateOverlayBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabStateOverlayBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabStateOverlayBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabStateOverlayBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FabStateOverlayBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabStateOverlayBackgroundDisabled", true);
+					}
+				}
```

Comment:
Should be inside of the `FloatingActionButton.Resources.Default` class

Reply:
Done.